be/src/olap/task/engine_clone_task.cpp

```diff
@@ -324,36 +251,24 @@ Status EngineCloneTask::_make_and_download_snapshots(DataDir& data_dir,
             remote_url_prefix = ss.str();
         }
 
-        st = _download_files(&data_dir, remote_url_prefix, local_data_path);
-        if (!st.ok()) {
-            LOG(WARNING) << "fail to download and convert tablet, remote=" << remote_url_prefix
-                         << ", error=" << st.to_string();
-            status = Status::InternalError("Fail to download and convert tablet");
-            // when there is an error, keep this program executing to release snapshot
-        }
+        status = _download_files(&data_dir, remote_url_prefix, local_data_path);
+        // when there is an error, keep this program executing to release snapshot
```

Comment:
Losing log here, should print log when download files failed.

Reply:
done